frontend/templates/troubleshooting/index.tsx

```diff
@@ -952,6 +953,7 @@ function RelatedProblems({
                   border="1px solid"
                   borderColor="gray.300"
                   borderRadius="md"
+                  overflow="hidden"
```

Comment:
Why is overflow hidden here? I don't think it's needed.

Reply:
Oh, it handles the border-radius!

Reply:
Yep, otherwise we get the bottom corner of the new flexbox extending out of the border radius.